weave/trace_server/calls_query_builder/calls_query_builder.py

```diff
@@ -213,8 +217,11 @@ def as_sql(
         cast: Optional[tsi_query.CastTo] = None,
     ) -> str:
         inner = super().as_sql(pb, "feedback")
-        param_name = pb.add_param(self.feedback_type)
-        res = f"anyIf({inner}, feedback.feedback_type = {param_slot(param_name, 'String')})"
+        if self.feedback_type == "*":
```

Comment:
would be nice to not in the interface definition that `feedback_type` can be a "*".

Reply:
Good point. Where is the interface defined though? I can add a comment on line 179 where the field is defined.

Reply:
ahh - i see it is not really made explicit anywhere. maybe add it to 380 of `weave/trace_server/trace_server_interface.py`